gtfs-realtime/CHANGES.md

```diff
@@ -17,29 +17,34 @@ When a producer or consumer is interested in adding a new field to the GTFS Real
 
 ### Specification amendment process
 1. Create a git branch with update of all relevant parts of protocol definition, specification and documentation files (except for translations).
-1. Create pull request on https://github.com/google/transit. Pull request must contain an extended description of the patch. The creator of the pull request becomes the _advocate_.
-1. Once pull request is registered, it must be announced by its advocate in the [GTFS Realtime mailing list](https://groups.google.com/forum/#!forum/gtfs-realtime). Once announced, the pull request is considered a proposal.
+2. Create pull request on https://github.com/google/transit. Pull request must contain an extended description of the patch. The creator of the pull request becomes the _advocate_.
```

Comment:
Please leave all these numbered as `1.` - Markdown automatically renumbers them in order when it renders them, and that way we don't have to manually manage the numbers.